### PR TITLE
Prepare Board documentation for new format docs

### PR DESCRIPTION
**Pull request by VitroidFPV**
As new target submissions will be *strongly encouraged* to provide documentations and/or board samples for developer verification + documentation, there should be a clear distinction between the old and the new pages.
image

The first entry takes from the provided example, more will be added as manufacturers/developers contribute docs for newly released (and maybe existing) boards.

### CODE REVIEW

**Comment by haslinghuis** [comment]
@VitroidFPV please note the board needs to be uppercase as detected by firmware. Also some boards like KAKUTE were recently updated - but perhaps not using new standard - if the intention is to start fresh.

**Comment by VitroidFPV** [comment]
Writing some instructions on how to submit the documentation as well, will push up in a little bit

**Comment by VitroidFPV** [comment]
I will rename it accordingly. The `Current` category serves to only show the documentation files that follow the new standard. I will try to set some time apart to write up the boards I have on hand

**Comment by haslinghuis** [comment]
Thank you - much appreciated - please check attached configurator PR to keep in sync.

**Comment by haslinghuis** [comment]
There is no new preview - so can not check without building local

image

Is the BOARD being duplicated in the path?

**Comment by VitroidFPV** [comment]
> Is the BOARD being duplicated in the path?

The document file itself is in a directory of the same name. That basically just allows a directory containing multiple files to show up as a single document in the sidebar - so that we can have images at the same directory level as the board documentation file without introducing a lot of clutter

**Comment by VitroidFPV** [comment]
Please review the documentation guide and recommend any potential changes. Other than that I think it's good to go 😅

**Review by haslinghuis — COMMENTED** [review]
LGTM

File: docs/development/manufacturer/fc_documentation/how-to-create-board-documentation.mdx [hunk 1]
```diff
@@ -0,0 +1,105 @@
+# How to Create Board Documentation
+
+The newly created board documentation format is made to take full advantage of the features available with the documentation platform in use - docusaurusVersion. It is extended with features not present in the default markdown format, and may be more complex to work with.
```
[the remark this answers is not in the document]
Comment (VitroidFPV) [reply docs/development/manufacturer/fc_documentation/how-to-create-board-documentation.mdx:3]: Autocomplete showing up where it really shouldn't 😅 I'll fix it and add a link to some markdown resources

**Comment by VitroidFPV** [comment]
> What is the difference between `Archive` and `Legacy`? i.e. should they be combined, or no?

Archive has board docs for boards that should still be supported, but don't follow the updated format. Legacy is for old F1/F3 boards. I thought about deleting those entirely, but this PR is just for organization

**Review by nerdCopter — APPROVED** [review]

**Review by haslinghuis — APPROVED** [review]

**Review by nerdCopter — APPROVED** [review]

**Comment by nerdCopter** [comment]
this repo properly dismisses approvals upon new commits. other repos do not, but such is convenient.

**Pull request merged**